### PR TITLE
Add prepush linting

### PR DESCRIPTION
**Pull request by Visya**
Added prepush linting and fixed remaining lining erros.

Now if the push fails, run `npm run lint:fix` to automatically fix errors, add fixes to commit and push.

### CODE REVIEW

File: components/HeaderLeft.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-  return (
-    <TouchableOpacity onPress={() => props.navigate('Home', {})}>
-      <View
+const HeaderLeft = props => (
+  <TouchableOpacity onPress={() => props.navigate('Home', {})}>
+    <View
+      style={{
+        paddingLeft: 16,
+        alignItems: 'center'
+      }}
```
Comment (albingroen) [inline components/HeaderLeft.js]: Prefer not to use inline-styling to make things more structured. Instead define styles in bottom of file and reference the style-name like this:

```javascript
style={styles.Button}

......

const styles = StyleSheet.create({
  Button: {
    padding: 15,
    borderRadius: 3,
    marginBottom: 50,
    backgroundColor: '#D26283',
    marginTop: 30,
    width: Dimensions.get('window').width - 40
  }
})
```

File: components/HeaderLeft.js [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+    >
+      <Text
         style={{
-          paddingLeft: 16,
-          alignItems: 'center'
+          color: '#fff',
+          fontSize: 18,
+          fontWeight: '600'
```
Comment (albingroen) [inline components/HeaderLeft.js]: Same here with styling.

File: components/HeaderRight.js [hunk 3]
```diff
@@ -2,27 +2,27 @@ import React from 'react'
...
-    </View>
-  )
-})
+const HeaderRight = props => (
+  <View
+    style={{
+      flexDirection: 'row',
+      alignItems: 'center',
+      paddingRight: 16
+    }}
```
Comment (albingroen) [inline components/HeaderRight.js]: Same here with styling.

File: components/HeaderRight.js [hunk 4]
```diff
@@ -2,27 +2,27 @@ import React from 'react'
...
+  >
+    <TouchableOpacity onPress={() => props.navigate('About', {})}>
+      <FontAwesome name='question-circle-o' size={32} color='white' />
+    </TouchableOpacity>
+    <TouchableOpacity onPress={() => props.navigate('InfoTicket', {})}>
+      <View
+        style={{
+          marginLeft: 16
+        }}
```
Comment (albingroen) [inline components/HeaderRight.js]: Same here with styling.

File: package.json [hunk 5]
```diff
@@ -7,7 +7,9 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "eject": "expo eject",
-    "test": "standard"
+    "lint": "standard",
+    "lint:fix": "standard --fix",
+    "prepush": "npm run lint"
```
Comment (albingroen) [inline package.json]: Why not also include to `--fix` on prepush?

Comment (Visya) [reply package.json]: after `--fix` the user needs to stage changes in Git and I wouldn't want to do some hack with creating a commit automatically

**Pull request merged**